WeakAuras/Transmission.lua

```diff
@@ -1675,6 +1675,18 @@ function WeakAuras.ImportString(str)
   end
 end
 
+function crossRealmSendCommMessage(prefix, text, chattype, target, queueName, callbackFn, callbackArg)
+  if chattype == "WHISPER" and target and not UnitIsSameServer(target) then
```

Comment:
What does UnitIsSameServer do for merged realms?

Reply:
i trusted https://wow.gamepedia.com/API_UnitIsSameServer on this, and i just did my own tests, it returns true when unit is on a linked server